packages/camera/tizen/src/camera_device.cc

```diff
@@ -343,41 +343,26 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
           [this](size_t width,
                  size_t height) -> const FlutterDesktopGpuBuffer * {
             std::lock_guard<std::mutex> lock(mutex_);
-            if (prepared_packet_ && !IsValidMediaPacket(prepared_packet_)) {
-              media_packet_destroy(prepared_packet_);
-              prepared_packet_ = nullptr;
-            }
-            if (current_packet_ && !IsValidMediaPacket(current_packet_)) {
-              media_packet_destroy(current_packet_);
-              current_packet_ = nullptr;
-            }
-            if (!prepared_packet_ && !current_packet_) {
+            if (!current_packet_) {
               return nullptr;
             }
-            if (prepared_packet_ && !current_packet_) {
-              current_packet_ = prepared_packet_;
-              prepared_packet_ = nullptr;
-            }
-
             tbm_surface_h surface = nullptr;
             int ret = media_packet_get_tbm_surface(current_packet_, &surface);
             if (ret != MEDIA_PACKET_ERROR_NONE) {
               LOG_ERROR("media_packet_get_tbm_surface failed, error: %d", ret);
-              media_packet_destroy(current_packet_);
```

Comment:
Why do you remove this line? if getting a tbm surface from media packets is fails, shouldn't media packets be destroyed? The reference is lost on the very next line.

Reply:
You're right. It seems to be a mistake! 👍